### DISCUSSION TITLE
Confirm In Modal

### DISCUSSION THREAD

**Discussion by hdahon** [wireui/wireui#635] [Q&A]
Hello,
Is it possible to call a confirm dialog from a modal ?
When i try  the confirm dialog always go under the modal.
<img width="971" alt="Capture d’écran 2023-03-22 à 15 39 18" src="https://user-images.githubusercontent.com/4508735/226940103-b394a621-fbac-43ee-a9de-544f36ee5b43.png">

Thanks

**Comment by rabol** [comment]
You might use a custom modal : https://livewire-wireui.com/docs/dialogs#custom-dialog and set the z-index value
